File: src/main/java/cn/lnkdoc/sdk/uia/instance/bjtoon/domain/AuthTicket.java
package cn.lnkdoc.sdk.uia.instance.bjtoon.domain;

import java.io.Serial;
import java.io.Serializable;

/**
 *
 * AuthTicket Object to be returned
 *
 * @author qinyi
 * @version  v 1.0 2018-10-23
 */
@SuppressWarnings(value = {"unused"})
public class AuthTicket implements Serializable {

	@Serial
	private static final long serialVersionUID = -50843L;
	/**
	 * authTicket
	 */
	private String authTicket;
	/**
	 * expiresIn
	 */
	private String expiresIn;

	/**
	 * getter
	 * 
	 * @return authTicket
	 */
	public String getAuthTicket() {
		return authTicket;
	}

	/**
	 * setter 
	 * 
	 * @param authTicket authTicket
	 */
	public void setAuthTicket(String authTicket) {
		this.authTicket = authTicket;
	}

	/**
	 * getter
	 * 
	 * @return expiresIn
	 */
	public String getExpiresIn() {
		return expiresIn;
	}

	/**
	 * setter
	 * 
	 * @param expiresIn expiresIn
	 */
	public void setExpiresIn(String expiresIn) {
		this.expiresIn = expiresIn;
	}


}

File: awesome-uia-sdk-support/awesome-uia-bjtoon-sdk/src/main/kotlin/cn/lnkdoc/sdk/uia/instance/bjtoon/domain/AuthTicket.kt
package cn.lnkdoc.sdk.uia.instance.bjtoon.domain

import java.io.Serializable


/**
 *
 * AuthTicket Object to be returned
 *
 * @author qinyi
 * @version  v 1.0 2018-10-23
 */
@Suppress("unused")
open class AuthTicket : Serializable {
    /**
     * authTicket
     */
    var authTicket: String? = null
    /**
     * expiresIn
     */
    var expiresIn: String? = null

    companion object {
        private const val serialVersionUID = -50843L
    }
}

